desktop/cmp/form/FormField.js

```diff
@@ -196,6 +213,19 @@ export class FormField extends Component {
             disabled: fieldModel && fieldModel.disabled,
             id: idAttr
         };
+
+        // Item should fill the available size of the FormField, unless dimensions are specified.
+        // Note: We explicitly set width / height to null to override defaults.
+        if (!item.props.width && !item.props.flex) {
```

Comment:
both the checks on 219 should be isUndefined() to allow zero and null settings to be respected